density/Documentation.md

```diff
@@ -1,13 +1,20 @@
+##Overview
+Columbia has shared with ADI a live stream of the nubmer of devices connected to routers at various locations on campus. New counts are added every fifteen minutes. 
+
 ###Get an API Key
-API keys are available for Columbia univserity affiliates with valid email addresses `uni@*.columbia.edu` `uni@barnard.edu`
+API keys are [available](density.adicu.com/auth) for Columbia univserity affiliates with valid email addresses `uni@*.columbia.edu` `uni@barnard.edu`
```

Comment:
the link for API keys must include `http://...`, otherwise github tries to make it a link to something within the repo